app/src/main/java/com/example/seminar_task1/HomeActivity.kt

```diff
@@ -2,10 +2,31 @@ package com.example.seminar_task1
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.widget.TextView
+import com.example.seminar_task1.databinding.ActivityHomeBinding
 
 class HomeActivity : AppCompatActivity() {
+    private lateinit var binding : ActivityHomeBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_home)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        initTransaction()
+    }
+
+    private fun initTransaction(){
+        val fragment1 = FollowerFragment()
+        val fragment2 = RepositoryFragment()
+
+        supportFragmentManager.beginTransaction().add(R.id.fcv_home,fragment1).commit()
```

Comment:
```suggestion
        supportFragmentManager.beginTransaction()
        .add(R.id.fcv_home,fragment1)
        .commit()
```
이렇게 쓰면 가독성이 더 좋아질 것 같습니다. :-) Android Studio 자체에 내장되어있는 `Reformat Code` 기능을 사용하면 아마 저렇게 바꿔줄 것입니다.